src/Event/ReadModel/Relations/Doctrine/DBALEventPlaceHistoryRepository.php

```diff
@@ -18,27 +18,25 @@ public function __construct(Connection $connection)
         $this->connection = $connection;
     }
 
-    public function storeEventPlaceStartingPoint(UUID $eventId, UUID $placeId): void
+    public function storeEventPlaceStartingPoint(UUID $eventId, UUID $placeId, DateTimeInterface $date): void
     {
-        $this->insertIntoPlaceHistoryTable($eventId, null, $placeId);
+        $this->insertIntoPlaceHistoryTable($eventId, null, $placeId, $date);
     }
 
-    public function storeEventPlaceMove(UUID $eventId, UUID $oldPlaceId, UUID $newPlaceId): void
+    public function storeEventPlaceMove(UUID $eventId, UUID $oldPlaceId, UUID $newPlaceId, DateTimeInterface $date): void
     {
-        $this->insertIntoPlaceHistoryTable($eventId, $oldPlaceId, $newPlaceId);
+        $this->insertIntoPlaceHistoryTable($eventId, $oldPlaceId, $newPlaceId, $date);
     }
 
-    public function insertIntoPlaceHistoryTable(UUID $eventId, ?UUID $oldPlaceId, UUID $newPlaceId): void
+    private function insertIntoPlaceHistoryTable(UUID $eventId, ?UUID $oldPlaceId, UUID $newPlaceId, DateTimeInterface $date): void
     {
-        $currentTimestamp = new DateTimeImmutable();
-
         $this->connection->insert(
             'event_place_history',
             [
                 'event' => $eventId->toString(),
                 'old_place' => $oldPlaceId ? $oldPlaceId->toString() : null,
                 'new_place' => $newPlaceId->toString(),
-                'date' => $currentTimestamp->format('Y-m-d H:i:s'),
+                'date' => $date->format('Y-m-d H:i:s'),
```

Comment:
I would use the exact same format as `recordedOn` and just call `toString` on it to store exactly the same value as inside the metadata.

Reply:
I am not using to String, I still use DateTimeInterface, but I did change all the formats to Atom. Is this oke?